Script/codecov_downloader.sh

```diff
@@ -2,5 +2,6 @@
 curl -s https://codecov.io/bash >codecov
 VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2)
```

Comment:
One small change is required here, should be `VERSION=$(grep 'VERSION=\"[0-9\.]*\"' codecov | cut -d'"' -f2);`